### ISSUE TITLE
Syntax error ?

### ISSUE THREAD

**Issue by rayjennings3rd** [kuestess/homebridge-platform-insteonlocal#163]
Tried installing this today and got:

ERROR LOADING PLUGIN homebridge-platform-insteon:
[10/3/2021, 6:19:54 PM] /usr/local/lib/node_modules/homebridge-platform-insteon/index.js:94
        var authtoken = "Bearer " + self.SecurityToken;
            ^^^^^^^^^

SyntaxError: Unexpected identifier
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Plugin.load (/usr/local/lib/node_modules/homebridge/src/plugin.ts:187:27)
    at /usr/local/lib/node_modules/homebridge/src/pluginManager.ts:126:16
[10/3/2021, 6:19:54 PM] ====================

Running MacOS, NodeJS 16.10

**Comment by kuestess** [comment]
@rayjennings3rd It looks like you’ve installed the wrong version of the plugin (`homebridge-platform-Insteon`), which requires a developer key.  If you want to use this plugin, then run `npm -g i homebridge-platform-insteonlocal`

**Comment by rayjennings3rd** [comment]
Thank you for your reply. I thought I looked at at the npm plugin and I thought (maybe based on the date) that there was not a lot of updates and that maybe it was an abandoned repo and the other one was the new one. I tried doing an: npm uninstall homebridge-platform-insteon and npm uninstall  npm -g i homebridge-platform-insteonlocal but I'm still getting that error after I restart homebridge. Am I not removing something properly? Thank you.
